amplitude/gradlew

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```

Comment:
Would be possible to move the gradlew wrapper (and all its components to the root folder?)

Reply:
On it